src/types.ts

```diff
@@ -20,9 +20,7 @@ export interface DefaultSettings {
 }
 
 export interface Settings extends Partial<DefaultSettings> {
-  publicKey?: string
-  fileName?: string
-  contentType?: string
+  publicKey: string
```

Comment:
А куда `fileName` и `contentType` делись?

Reply:
I decided to remove this props, because this is little strange setup filename with uploader constructor.

like:

```
const uploader = new UploadClient({ fileName: 'default-filename' })
```

but ofc it can be helpful for setup default file name and content type in node
I'm going to return this props